apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/ToggleMinOppgaveliste.tsx

```diff
@@ -24,7 +24,9 @@ const TabsWidth = styled(Tabs)`
 
 export const ToggleMinOppgaveliste = () => {
   const innloggetSaksbehandler = useAppSelector((state) => state.saksbehandlerReducer.innloggetSaksbehandler)
-
+  if (!innloggetSaksbehandler.skriveTilgang) {
+    return null
+  }
```

Comment:
Skal ikke saksbehandler uten skrivetilgang få se oppgavelisten? 🤔

Reply:
Riktig

Reply:
Altså alt er up for discussion I guess, men mitt beste guesstimate basert på litt samtale med NKS osv